pymatgen/io/abinit/abiobjects.py

```diff
@@ -720,14 +720,9 @@ def __init__(
             ngkpt = np.reshape(kpts, 3)
             shiftk = np.reshape(kpt_shifts, (-1, 3))
 
-            if use_symmetries and use_time_reversal:
-                kptopt = 1
-            if not use_symmetries and use_time_reversal:
-                kptopt = 2
-            if not use_symmetries and not use_time_reversal:
-                kptopt = 3
-            if use_symmetries and not use_time_reversal:
-                kptopt = 4
+            kptopt = 1 if use_symmetries else 2
+            if not use_time_reversal:
+                kptopt += 2
```

Comment:
I think this is wrong now.
If use_symmetries and not use_time_reversal: 4 in old verdion but 3 in the new one.

Reply:
Yes you are right! Thanks for catching this. I would revert this now.